add CMEK info about existing instances and Azure keys (#606)
neo4j/docs-aura: modules/ROOT/pages/security/encryption.adoc

. In *Select members*, paste the *Neo4j CMK Application name* that is displayed in the Aura Console.
. The *Neo4j CMK Application* should appear, select this application then *Review + Assign*.

=== Azure key rotation

> If you rotate an Azure key and immediately disable the old one, the connection status in Aura changes from "Ready" to "Pending".

Reviewer: I'd suggest the following change:

```suggestion
If you immediately disable the old key version after the Azure key is rotated, the connection status in Aura changes from "Ready" to "Pending".
```

Since the key can be rotated automatically.
